config/database.yml.distr

```diff
@@ -27,3 +27,18 @@ production:
     migrations_paths: db/cdr_migrate
     #min_messages: warning
     schema_cache_path: tmp/cdr_schema_cache.yml
+  cdr_replica:
+    adapter: postgresql
+    encoding: unicode
+    database: cdr
+    pool: 5
+    username: yeti
+    password: somepassword
+    host: 127.0.0.1
+    schema_search_path: 'cdr, reports, billing, public'
+    port: 5432
+    migrations_paths: db/cdr_migrate
```

Comment:
Do we really need **migrations_paths** and  **schema_cache_path** configuration for replica?

Reply:
@BigG1947 @dmitry-sinina 
- migrations_paths - should be removed
- schema_search_path - not sure, looks like we have to keep it for replica

Reply:
@BigG1947 please remove migrations_paths/schema_cache_path for replica configuration example